Makefile

```diff
@@ -25,3 +25,23 @@ coverage:
 coveralls: coverage
 	go get github.com/mattn/goveralls
 	goveralls -coverprofile=$(COVERAGE_FILE) -service=github
+
+.PHONY: bench
+bench:
+	go clean -testcache
+	go test -bench=. -benchmem -benchtime=1s
+
+.PHONY: ab-perf
+ab-perf:
+	export TNT_CUR_BRANCH=$(git branch --show-current)
+	go clean -testcache
+	go test -bench=. -benchmem -benchtime=1s > b.txt
+	git stash && git fetch && git checkout master
```

Comment:
I don't like unexpected manipulations with my working directory. How about cloning `master` into a gitignored or temporary directory and keep the working directory untouched?

Reply:
Fixed.

Reply:
BTW, we can be interested in `$(git merge-base master..HEAD)`, not just `master` (which may go ahead).

Reply:
Ping regarding `git merge-base`.